internal/rgw/policy.go

```diff
@@ -0,0 +1,60 @@
+package rgw
+
+import (
+	"context"
+
+	awss3 "github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+	"github.com/linode/provider-ceph/apis/provider-ceph/v1alpha1"
+	"github.com/linode/provider-ceph/internal/backendstore"
+	"github.com/linode/provider-ceph/internal/otel/traces"
+	"go.opentelemetry.io/otel"
+)
+
+const (
+	errGetBucketPolicy    = ""
+	errPutBucketPolicy    = ""
+	errDeleteBucketPolicy = ""
```

Comment:
Missing values.

Reply:
Added in https://github.com/linode/provider-ceph/pull/233/commits/48d5d4bdbd158135087c43cafc7f78d2c7d0f4d0